src/components/slider.tsx

```diff
@@ -0,0 +1,88 @@
+import { getTrackBackground, Range } from "react-range"
+import React, { FC, useState } from "react"
+
+import InputFilter from "./filters/input"
+import Label from "./fieldHelpers/label"
+import colors from "../tailwind/colors"
+
+interface Props {
+  step: number
+  min: number
+  max: number
+  name: string
+  defaultValue: number
+  inputLabel: string
+  required?: boolean
```

Comment:
we need an `onChange` handler here because otherwise we cannot control it with formik